pkg/cortex/cortex.go

```diff
@@ -66,15 +66,15 @@ type Config struct {
 	Ingester       ingester.Config          `yaml:"ingester,omitempty"`
 	Storage        storage.Config           `yaml:"storage,omitempty"`
 	ChunkStore     chunk.StoreConfig        `yaml:"chunk_store,omitempty"`
-	Schema         chunk.SchemaConfig       `yaml:"schema,omitempty"`
+	Schema         chunk.SchemaConfig       `yaml:"schema,omitempty" doc:"hidden"`
```

Comment:
Why are these config options hidden?

Reply:
I totally forgot to mention. TL;DR the `SchemaConfig` can't be automatically generated and my plan is to manually document it in a subsequent PR.

Why can't be generated? Because this tool is based on a mapping between the config structs and CLI flags. We have flags registered for the `LegacySchemaConfig`, but not for the `PeriodConfig`. Moreover the `PeriodConfig` is a slice and slices of struct - in the config - don't play nicely with this tool due to how CLI flags are mapped with the config (by memory address, but a default slice would be empty so there's nothing to map even if we would have the CLI flags).

Reply:
Ack, could you leave a comment in the code and then lets get this merged!

Reply:
Commented.

> then lets get this merged!

Let's wait for the CI to pass. Please also note that there's only 1 maintainer approval.

Reply:
> Please also note that there's only 1 maintainer approval.

I'm assuming you approve of your own PR ;) It's okay if you don't :P